### PR TITLE
Bug: General fixes

### PR DESCRIPTION
**Pull request by mzegar**
Just some general fixes / changes before we release.

### CODE REVIEW

File: code/player/camera/SpectateCamera.cs [hunk 1]
```diff
@@ -71,6 +71,21 @@ private Vector3 GetViewOffset()
...
+            foreach (Client client in Client.All)
+            {
+                if (client.Pawn is TTTPlayer player && player.LifeState == LifeState.Alive)
+                {
+                    players.Add(player);
+                }
+            }
+
+            return players;
+        }
```
Comment (Alf21) [inline code/player/camera/SpectateCamera.cs:88]: Could you put it into Player.cs?
Could you modify <https://github.com/TTTReborn/ttt-reborn/blob/main/code/player/Player.cs#L36-L49> as well to use `Client.All`?
Does it work for client and server? (so Clients know who is alive)?

**Review by Alf21 — APPROVED** [review]

File: code/player/camera/SpectateCamera.cs (hunk 1 above)
Comment (Alf21) [reply code/player/camera/SpectateCamera.cs:88]: > Does it work for client and server? (so Clients know who is alive)?

Just tested, indeed, it is... That's bad :D

Comment (Alf21) [reply code/player/camera/SpectateCamera.cs:88]: Gonna add this method in the scoreboard PR

Comment (mzegar) [reply code/player/camera/SpectateCamera.cs]: Are we fine with my changes? I think it'll be better just to start spinning off any util methods we need.

**Review by Alf21 — APPROVED** [review]

**Pull request merged**